### ISSUE TITLE
[FEATURE] Containerization

### ISSUE THREAD

**Issue by Zeighnab** [FreeOps-Tools/LogIQ#31]
### Description

Containerized the application into portable software so as to run on various systems.

**Comment by Zeighnab** [comment]
I would love to be assigned to this

**Comment by rufilboy** [comment]
It has been assigned to you! Thanks for your interest in open source

**Comment by Zeighnab** [comment]
Yes, i amOn 17 Oct 2023, at 2:38 PM, Ilyas Rufai ***@***.***> wrote:﻿
@Zeighnab are you still interested in working on this?

**Comment by rufilboy** [comment]
Okay, will be expecting your update before the end of the month

**Comment by rufilboss** [comment]
@Zeighnab are you still working on this??
